### ISSUE TITLE
Unhandled exceptions with empty params passed

### ISSUE THREAD

**Issue by dchertousov** [Travelport-Ukraine/uapi-json#247]
If no params passed into `createAirService`, it falls with
```
TypeError: Cannot read property 'auth' of undefined
```

If params is passed as not an object, or empty object (`{}`), it falls with
```
TypeError: Cannot read property 'region' of undefined
```

It's hard to test dependant modules without typed error raised.

**Comment by dchertousov** [comment]
Duplicates #163

**Comment by dchertousov** [comment]
duplicate of #163

**Comment by dchertousov** [comment]
Closed by #265
